src/tests/calculate.test.js

```diff
@@ -0,0 +1,88 @@
+import calculate from "../logic/calculate";
+
```

Comment:
We need to write a unit test for all the functionalities & scenarios in `calculate.js` module.

Example:
Kindlly write unit test for the following:
1. `buttonName === '.'`
2. `buttonName === '='`

Reply:
We already did that. It's in a file called `calculate.test.js`. Hard to miss!

Reply:
Kindly double-check, as you can see, I drop a comment on `calculate.test.js`.

Reply:
Done.

Reply:
@crgc you deleted your first comment 😄, it would be nice to be professional with our comment 👍, thank you.